src/utils/utils.js

```diff
@@ -162,16 +170,16 @@ const PATH_END_REASONS = {
  * @param {boolean} deepOnly
  * @param {boolean} inline
```

Comment:
I'm aware the docstring was already missing documentation for `inScope` but could you document `stopFunction`?